docs/contributing.md

```diff
@@ -286,7 +286,7 @@ the literal suffix.
 ### New source file template
 
 If you are adding new source file, there is a template you should use.
-Specifically, every source file should start with the licence header:
```

Comment:
I am reasonably sure that "licence header" is correct for British english.

Reply:
Reverted.